modules/consensus/applytransaction_test.go

```diff
@@ -427,7 +427,7 @@ func TestMisuseApplyFileContractRevisions(t *testing.T) {
 	txn := types.Transaction{
 		FileContractRevisions: []types.FileContractRevision{{}},
 	}
-	cst.cs.applyFileContractRevisions(bn, txn)
+	cst.cs.applyFileContractRevisions(pb, txn)
```

Comment:
such a huge fraction of this diff set is just 'bn' -> 'pb'.

I was afraid of this PR but it's actually not so bad.